server/data/allocationManagerApiClient.ts

```diff
@@ -0,0 +1,19 @@
+import RestClient from './restClient'
+import config from '../config'
+import { Pom } from '../interfaces/pom'
+
+export default class AllocationManagerClient {
```

Comment:
This should be named `AllocationManagerApiClient` to match the filename, and also needs to implement `AllocationManagerClient`